### PR TITLE
Eventbrite Block: Add isDefault to the Embed button

### LINKED ISSUE #14462: Eventbrite block: Improve default UI
This issue tracks two different enhancements:
- [x] ~Provide a default text (e.g. "Register") for the block's Button mode.
Right now, a missing text for the button could go unnoticed in the editor where it has a placeholder text, but looks weird on the front end.~ #14585

- [x] ~Return a link to the event on the front end.
When reading the post in contexts that don't have the Eventbrite block (mail, feeds, with Jetpack deactivated) the block would still show something useful.~ #14510

- [x] ~Add default styling to the Embed button in the placeholder so that it has a border like all other blocks.~ #14588

### PR DESCRIPTION
**Pull request by apeatling**
Partially fixes #14462

#### Changes proposed in this Pull Request:

Make sure the embed button has a border by using `isDefault` so it matches all other embed blocks.

**Before:**

Screen Shot 2020-02-05 at 3 53 50 PM

**After:**

Screen Shot 2020-02-05 at 3 53 04 PM

#### Testing instructions:
* Insert an Eventbrite block
* Confirm the Embed button has a border

### CODE REVIEW

**Comment by Copons** [comment]
This works for me _right now_, but it will have problems in the future.

In https://github.com/WordPress/gutenberg/pull/19058, the button component's `isDefault` prop has been deprecated in favour of `isSecondary`.

https://github.com/WordPress/gutenberg/blob/164d5830a9acd895dfd661b5fde1ca8b80b270ac/packages/components/src/button/index.js#L48-L52

https://github.com/WordPress/gutenberg/blob/164d5830a9acd895dfd661b5fde1ca8b80b270ac/packages/components/src/button/index.js#L55

This means that:
- Until Gutenberg 7.2: `isDefault` works, and `isSecondary` doesn't.
- After 7.2: `isSecondary` works, and `isDefault` works too, but is marked as deprecated (throwing a console warning).

If I'm not mistaken, WPCOM uses 7.3 (but 7.4 just released yesterday), and I honestly have no idea how to check which version Core WP uses (I think <7.2 though, as WP 5.3 has been released _before_).

In other words: if we only add `isDefault`, the button works now, but won't in the future.
If we only add `isSecondary`, the button won't work now, but will in the future.
The solution seems to be: let's add both, and maybe have a `@todo` comment explaining that we might want to remove `isDefault` whenever the minimum WP version supported by Jetpack uses Gutenberg >7.2.

This also affects `isLarge`, which changes the button size on <7.2, but does literally nothing on later versions.
Again: let's leave it with a comment explaining that it will become useless in the future.

**Comment by apeatling** [comment]
Thanks for the information @Copons, that's great to know. I've added `isSecondary` and left a `@todo` comment to mention removing in the future.

File: extensions/blocks/eventbrite/edit.js [hunk 1]
```diff
@@ -273,7 +273,9 @@ class EventbriteEdit extends Component {
 							placeholder={ __( 'Enter an event URL to embed here…', 'jetpack' ) }
 							onChange={ event => this.setState( { editedUrl: event.target.value } ) }
 						/>
-						<Button isLarge type="submit">
+						// @todo Remove isDefault and isLarge when the minimum WP version supported by Jetpack
+						uses Gutenberg > 7.2
```
Comment (Copons) [inline extensions/blocks/eventbrite/edit.js]: Missing some `//`? 😄

**Review by Copons — DISMISSED** [review]
This LGTM (after fixing the missing comment)! ✨

File: extensions/blocks/eventbrite/edit.js (hunk 1 above)
Comment (apeatling) [reply extensions/blocks/eventbrite/edit.js]: Oh, weird. I added this all on one line, but on commit it was cut into two without the extra `//`?

**Comment by apeatling** [comment]
Missing comment fixed. :)

**Comment by apeatling** [comment]
Wait, no. Same thing happened again with that commit. I'll dig in further.

**Review by jeherve — APPROVED** [review]

**Comment by jeherve** [comment]
Cherry-picked to `branch-8.2` in acaf37834923ff956258f1df5b08d80f481fad0a

**Comment by jeherve** [comment]
r202642-wpcom
